client/src/app/site/pages/meetings/services/media-manage.service.ts

```diff
@@ -68,10 +56,15 @@ export class MediaManageService {
     }
 
     public getLogoUrl(place: LogoPlace): string | null {
-        // Note: we are not fetching the mediafile view model at any place.
-        const mediafileId = this.activeMeetingService.meeting?.logo_id(place);
+        // Note: we are not fetching the mediafile view model at any place except when filtering for the defaults.
+        const mediafileId = this.activeMeetingService.meetingId
+            ? this.activeMeetingService.meeting?.logo_id(place)
+            : null;
+        const generalMediafileId = this.getGlobalMediafileIdByToken(place);
```

Comment:
Move this into the if-else to only search for the token file if no meeting is active